app/controllers/airlines_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AirlinesController < ApplicationController
+  def show
+    @airline = Airline.find(params[:id])
+    @adult_passengers = @airline.adult_passengers.distinct
```

Comment:
Why are you calling `.distinct` here?

Reply:
I have such a bad habit of leaving things where they should not be